site/soc/software/config/versions.yaml

```diff
@@ -8,6 +8,13 @@ data:
         location: /armada/airship-components/openstack-helm-infra
         subpath: ingress
     osh:
+      # TODO remove after Treasuremap catches up on uplifting version
+      horizon:
+        location: https://git.openstack.org/openstack/openstack-helm
+        #reference: 38fb90d52824f9d23c99aab91e23097cd9b657d9
```

Comment:
Should we remove that commented version ?

Reply:
yes :-)

Reply:
removed.